libs/shared/l10n/src/lib/localizer/localizer.rsc.factory.spec.ts

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Test, TestingModule } from '@nestjs/testing';
+import { LocalizerRscFactory } from './localizer.rsc.factory';
+import { ILocalizerBindings } from './localizer.interfaces';
+import supportedLanguages from '../supported-languages.json';
+import { LocalizerRSC } from './localizer.rsc';
```

Comment:
nit: why `LocalizerRSC` (with uppercases) but then `LocalizerRscFactory` (with lowercases)?

Reply:
I asked myself that same thing. Ended up sticking with `*Rsc*`. TY :)